### PR TITLE
chore: adjust store calls to database

### PR DESCRIPTION
**Pull request by alexandreferris**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

This PR is to handle the changes from Kalium when storing the calls in the database.

### Dependencies (Optional)

Needs releases with:

- [X] [#686 chore: add calls to database](https://github.com/wireapp/kalium/pull/686)

### Testing

Tests were done in code and also with @dogaguclu

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallViewModel.kt [hunk 1]
```diff
@@ -31,20 +32,24 @@ class InitiatingCallViewModel @Inject constructor(
     savedStateHandle: SavedStateHandle,
     private val navigationManager: NavigationManager,
     private val conversationDetails: ObserveConversationDetailsUseCase,
-    private val allCalls: GetAllCallsWithSortedParticipantsUseCase,
+    private val observeEstablishedCallsUseCase: ObserveEstablishedCallsUseCase,
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallViewModel.kt:35]:
```suggestion
    private val observeEstablishedCalls: ObserveEstablishedCallsUseCase,
```

**Review by yamilmedina — APPROVED** [review]
great work ! A small adjustment is needed, to comply with the rest of the naming

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-778/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-778/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-778/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-778/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3926166)(3926166)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3926166)(3926166)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-778/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3934140)(3934140)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3934140)(3934140)-dev-release.apk)

**Pull request merged**